base/services/wuauserv/wuauserv.c

```diff
@@ -0,0 +1,130 @@
+/*
+* PROJECT:          ReactOS Services
+* LICENSE:          GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+* PURPOSE:          Automatic Updates service stub.
+*                   Stub is required for some application at the installation phase.
+* COPYRIGHT:        Copyright 2018 Denis Malikov (filedem@gmail.com)
+*/
+
+#include "wuauserv.h"
+
+/* GLOBALS ******************************************************************/
+
+static WCHAR ServiceName[] = L"wuauserv";
+
+static SERVICE_STATUS_HANDLE ServiceStatusHandle;
+static SERVICE_STATUS ServiceStatus;
+
+static HANDLE exitEvent = NULL;
+
+/* FUNCTIONS *****************************************************************/
+
+static VOID
+UpdateServiceStatus(DWORD dwState)
+{
+    ServiceStatus.dwServiceType = SERVICE_WIN32_OWN_PROCESS;
+    ServiceStatus.dwCurrentState = dwState;
+    ServiceStatus.dwControlsAccepted = 0;
+    ServiceStatus.dwWin32ExitCode = 0;
+    ServiceStatus.dwServiceSpecificExitCode = 0;
+    ServiceStatus.dwCheckPoint = 0;
+
+    if (dwState == SERVICE_START_PENDING ||
+        dwState == SERVICE_STOP_PENDING ||
+        dwState == SERVICE_PAUSE_PENDING ||
+        dwState == SERVICE_CONTINUE_PENDING)
+        ServiceStatus.dwWaitHint = 10000;
+    else
+        ServiceStatus.dwWaitHint = 0;
+
+    SetServiceStatus(ServiceStatusHandle,
+                     &ServiceStatus);
+    DPRINT1("WU UpdateServiceStatus() called\n");
+}
+
+static DWORD WINAPI
+ServiceControlHandler(DWORD dwControl,
+                      DWORD dwEventType,
+                      LPVOID lpEventData,
+                      LPVOID lpContext)
+{
+    DPRINT1("WU ServiceControlHandler() called\n");
```

Comment:
If you move the name of the function in the DPRINT's in the switch statement, there is just one line in the log per call to this function, instead of there always being 2 lines.